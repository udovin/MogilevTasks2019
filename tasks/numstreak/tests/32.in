162744956115086736 10
