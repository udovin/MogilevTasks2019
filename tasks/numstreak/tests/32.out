58710301628820
